addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -389,9 +403,9 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
                 self.leaderBoard.showLeaderboard(false, false);
             });
         } else {
-            self.$('.o_survey_session_close').click();
-            self.$('.o_survey_session_results').addClass('d-none');
-            self.$('.o_survey_session_description_done').prepend($("<h1>").text(_t('Thank you!'))).removeClass('d-none');
+            // There are multiple o_survey_session_close in the page so :first is mandatory
+            self.$('.o_survey_session_close:first').click();
+            self._displaySessionClosedPage();
```

Comment:
Do you mean to use `first-of-type`?
Furthermore, I think the next line is self explanatory, I don't see why else you would use it.
Finally, maybe not necessary depending on the answer to the previous comment?

Reply:
Yes, good catch, thank you. While I found some tutorial with the selector :first, it is not listed in w3c. 
Actually, it was a bug, as there are 2 such element on the page, the session was closed twice leading to some errors. I just need to select one of the element on the page. As I didn't find such selector, I just used the method first() from jquery.